src/components/common/TmBalance.vue

```diff
@@ -41,14 +41,15 @@
   </div>
 </template>
 <script>
-import { shortDecimals } from "scripts/num"
 import refetchNetworkOnly from "scripts/refetch-network-only"
+import { shortDecimals } from "scripts/num"
 import { noBlanks } from "src/filters"
-import TmBtn from "common/TmBtn"
-import SendModal from "src/ActionModal/components/SendModal"
-import ModalWithdrawRewards from "src/ActionModal/components/ModalWithdrawRewards"
 import { UserTransactionAdded } from "src/gql"
 import { mapGetters } from "vuex"
+const TmBtn = () => import("common/TmBtn")
+const SendModal = () => import("src/ActionModal/components/SendModal")
```

Comment:
let's load modals async and make sure they are not rendered directly (v-if) but let's remove the noise for the other components?